src/runtime/classobject.cs

```diff
@@ -143,7 +144,7 @@ public override IntPtr type_subscript(IntPtr idx)
                 return Exceptions.RaiseTypeError("type(s) expected");
             }
 
-            Type gtype = AssemblyManager.LookupType($"{type.FullName}`{types.Length}");
+            Type gtype = AssemblyManager.LookupTypes($"{type.FullName}`{types.Length}").FirstOrDefault();
```

Comment:
Picking the first type of many randomly is not a good solution.

Reply:
That's true, but this is what we are doing right now as well. I can add a follow-up ticket, but I also want to get to a warning-free build at some point s.t. we can make them fatal.

Reply:
That would do.

Reply:
#1320.